rotkehlchen/data/importer.py

```diff
@@ -182,3 +186,183 @@ def import_cointracking_csv(self, filepath: Path) -> None:
                     continue
 
         return None
+
+    def _consume_cryptocom_entry(self, csv_row: Dict[str, Any]) -> None:
```

Comment:
While reading this function I see it has become huge and also perhaps some of the parsing logic is similar between if branches. Can you maybe split it into smaller functions?

Reply:
Let's move this requedt to another PR as discussed in discord.